### DISCUSSION TITLE
Join the Serverpod Community on Discord! (OFFICIAL)

### DISCUSSION THREAD

**Discussion by Katekko** [serverpod/serverpod#2318] [Ideas]
##  A New Home for Serverpod Developers 🤝

Calling all Serverpod enthusiasts!  While we eagerly await an official community hub, I've taken the initiative to create a dedicated Discord server for developers to connect, collaborate, and learn together.

**Why Join?**

* **Connect with fellow developers:**  Share your projects, ask questions, and build valuable connections with other Serverpod users.
* **A space for learning:**  Learn from each other's experiences, discover new techniques, and grow your Serverpod skills.
* **Stay informed:**  Receive updates on the latest Serverpod news, events, and resources.

**Join us!**  [Serverpod Discord Server](https://discord.gg/wJ4pQeHhVc)

**Comment by SandPod** [comment]
Hey @Katekko!

Now that our official [Serverpod Discord Server](https://discord.gg/wJ4pQeHhVc) is launched, perhaps we should update this post to the official Serverpod one? Would you like to perform the edit?

Would also be great to combine the communities to find even more Serverpod developer synergies. Let me know if I can help out with any migration :)

**Reply by Katekko** [reply]
For sure, updating now, I posted the official discord in the community aswell

See if the update was okay, if not, you can edit what is missing

**Reply by SandPod** [reply]
This looks great! Thank you 🚀 

And thank you for taking the initiative to connect fellow Serverpod enthusiasts before the official server was launched.
